verifier/src/lib.rs

```diff
@@ -168,22 +185,111 @@ pub enum VerificationError {
     },
 }
 
-impl DisplayableError for VerificationError {}
+impl ResultMessage for VerificationResultMetadata {
+    fn fmt_success(&self, f: &mut Formatter, pad: usize) -> core::fmt::Result {
+        let status = SUCCESS_MESSAGE_INDICATOR;
+        match self {
+            Self::General => write!(f, "{:pad$}{status} Success", "")?,
+            Self::AlwaysFalse => write!(f, "{:pad$}{status} How did you get here", "")?,
```

Comment:
For all of this code coverage I'm fine making tests for these, IFF we decide this is the correct path to go down.

Reply:
refactored comment no longer relevant